test/util/utils.sh

```diff
@@ -27,3 +27,70 @@ function kubeconfig_cleanup {
     kubectl config delete-user "${cluster_name}-admin" || true
     kubectl config delete-cluster "${cluster_name}" || true
 }
+
+function collect_standalone_cluster_diagnostics {
+    cluster_infra=$1
+    cluster_name=$2
+
+    if [[ -z "${cluster_infra}" ]]; then
+        echo "Cluster infra (vsphere, azure, aws, docker) not passed to collect_standalone_cluster_diagnostics function. Usage example: collect_standalone_cluster_diagnostics vsphere test-cluster-1234"
+        return 1
+    fi
+
+    if [[ -z "${cluster_name}" ]]; then
+        echo "Cluster name not passed to collect_standalone_cluster_diagnostics function. Usage example: collect_standalone_cluster_diagnostics vsphere test-cluster-1234"
+        return 1
+    fi
+
+    echo "Collecting ${cluster_name} standalone cluster diagnostics data"
+
+    tanzu diagnostics collect --workload-cluster-infra "${cluster_infra}" \
```

Comment:
There are just a few things we run in the diagnostics collect function, mainly the `tanzu diagnostics collect` command runs and if that fails - what do we do is the question

Reply:
If we fail to get logs, it's no better than it is today. We should just display the error for someone to attempt to address why the logs aren't being collected.

Reply:
Cool, then I'll add a `|| true` to absorb any errors in the log collection command